### PR TITLE
feat: add support for `jsconfig.json`

### LINKED ISSUE #73: Support for jsconfig.json
Our project doesn't use Typescript by rather only Javascript, thus only have `jsconfig.json` file. Does this package support `jsconfig.json`? Thanks!

### PR DESCRIPTION
**Pull request by rbong**
close #73 

tl;dr - this handles the case of `project: "path/to/jsconfig.json"` without breaking `project: "/path/to/dir"`.

### Context: `get-tsconfig` behaviour

If a path other than `tsconfig.json` is used, `getTsconfig` requires that the first argument is a directory and the second argument is a path.

The second argument to `getTsconfig` must not be a directory, otherwise it will try to read it as a file, resulting in an error.

### Current behaviour

The given `project` paths, or the resulting paths from globs, are passed as the first and only argument to `getTsconfig`.

### Changes

In the case that the given `project` path is a file, or a glob results in a file, its parent directory will be passed as the first argument to `getTsconfig` and its base name will be the second argument.
This fixes passing full filenames other than `tsconfig.json`, ex. `jsconfig.json`.

In the case that a given `project` path is a directory, or a glob results in a directory, the full path will be passed as the first and only argument to `getTsconfig`.
This distinction is necessary, since if the base name of a directory is passed as the second argument it will result in an error.

### CODE REVIEW

**Comment by rbong** [comment]
My mistake - didn't check coverage. Adding another test case now.

**Comment by JounQin** [comment]
It seems you didn't follow own commit message lint at the same time.

See also https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional

**Comment by rbong** [comment]
@JounQin thanks, I was having some issues with my editor so it didn't run. Checks are passing now.

**Review by JounQin — APPROVED** [review]

**Comment by JounQin** [comment]
@rbong Thanks for your contribution!

**Comment by JounQin** [comment]
It seems I/we forgot to update README to mention this. 🤣
